lib/errors/ErrorPage.js

```diff
@@ -1,16 +1,32 @@
 "use strict";
 
-var ErrorViews = require("./ErrorViews");
+var DEFAULT_ERROR_CODE = "500";
 
-var ErrorViewMapping = {
+var ErrorPage = {
 
     create: function(mapping) {
-        return new ErrorViews(mapping);
+        return mapping;
```

Comment:
this change is here to maintain backward compatibility for this major version while introducing the new functionality. this code will be removed in version 2.x